README.md

```diff
@@ -60,6 +60,25 @@ HARRY HALPIN Research Scientist at Massachusetts Institute of Technology [↩](#
 
 ## Cryptoeconomy KYODO
 
+### KYODO Impression revenue model
+> Now that the commodity value of human labor is becoming increasingly tangential in an even more automated world, new ways of defining human worth and the social relationship will need to be explored
+>
+> -- *Jeremy Rifkin* "End of work"
+
+"Pay per hour" or monthly salary better fits for robots that produce countable units with a stable quality. 
+How the enthusiasm or dedication could be evaluated?
+How the progressiveness or intuition could be estimated?
+How the single time but relevant contribution could be rewarded?
+
+KYODO suggests Impression revenue model. The contribution is evaluated by members of the community in the momentum. The Nash Equilibrium is achieved through the Token economy, where the token value is based on the activity of the members. Means that every member of a community is interested to evaluate other contributors fair and motivate them to further actions. Also, the group members know the best if someone did a good job. 
```

Comment:
Maybe token value is based on the effectiveness of the activity of the members?
Actually as it is asset backed the price is based on the cap of the fund, but holders/members interests are aligned to increase the token price by increasing its cap

Reply:
agree